### PR TITLE
Contributing Guidelines

### PR DESCRIPTION
**Pull request by anneschuth**
Adds contributing guidelines, a HowTo and a default code of conduct.

Curious to hear your thoughts.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,3 +34,7 @@ metadata such as the number of words and named entities from the text.
 >>> print(pipe(sample_text))
 ("clean_text":'Sample text!', "language": 'en', "nwords": 2)
 ```
+
+
+## Contributing
+See [CONTRIBUTING](CONTRIBUTING) for guidelines for contributers.
```
Comment (lmdehaas) [inline README.md]: CONTRIBUTING.md? This link is broken.

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+confidentiality with regard to the reporter of an incident.
+Further details of specific enforcement policies may be posted separately.
+
+Project maintainers who do not follow or enforce the Code of Conduct in good
+faith may face temporary or permanent repercussions as determined by other
+members of the project's leadership.
+
+## Attribution
+
+This Code of Conduct is adapted from the [Contributor Covenant][homepage], version 1.4,
```
Comment (lmdehaas) [inline CODE_OF_CONDUCT.md]: Link hasn't been filled in yet

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
+# Contributing
+Textpipe is open source, everyone is free to contribute. 
+But please follow the guidelines below.
+
+## Guidelines
+### Code of conduct
+Please follow our [code our conduct](Contributor Covenant Code of Conduct).
```
Comment (lmdehaas) [inline CONTRIBUTING.md]: As above, link hasn't been filled in yet

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+
+### Coding Conventions
+All code (except for deployment scripts) is in Python 3.6.
+We follow the [Google Python Style Guide](https://github.com/google/styleguide/blob/gh-pages/pyguide.md) which means that we use `pylint`.
```
Comment (dodijk) [inline CONTRIBUTING.md]: Maybe we can make this a bit stronger?
> that we check for style using `pylint` and enforce compliance without warnings and errors?

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+### Testing
+All (new) methods and functions have to be unittested using Python `doctests`.
+
+### Reviewing
+All changes (not matter how minor) to master are made through Pull Requests (PRs). 
+All PRs are reviewed by at least one reviewer. 
+If you know that someone may have an opinion on your change, make sure this person reviewed your code before merging.
+
+### CI/CD
+We run a Continues Integration (CI) pipeline on Travis. Travis runs `script/test`, which in turn runs all tests in the `tests` folder and all `doctests`.
```
Comment (dodijk) [inline CONTRIBUTING.md]: continuous?

Comment (dodijk) [reply CONTRIBUTING.md]: [Travis](http://travis-ci.org)

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+   $ git push -u origin my-feature
+   ```
+
+   * **Be descriptive in your commit messages.**
+   * **Commit changes that belong together.**
+
+4. Browse to [https://github.com/textpipe/textpipe](https://github.com/textpipe/textpipe) and follow instructions to create Pull Request. 
+   Make sure you add reviewers to your PR. Your code should be reviewed by at least 1 person and by everyone you know could have an opinion on your change.
+   
+5. Use [Slack](https://textpipe.slack.com/signup) to advertise your PR.
```
Comment (dodijk) [inline CONTRIBUTING.md]: (Optionally)?

File: CODE_OF_CONDUCT.md (hunk 2 above)
Comment (anneschuth) [reply CODE_OF_CONDUCT.md]: it's a footnote

**Comment by anneschuth** [comment]
@dodijk I've added contributers

**Pull request merged**